src/main/java/com/dao/impl/CodeMySQLDaoImpl.java

```diff
@@ -0,0 +1,81 @@
+package com.dao.impl;
+
+import com.dao.CodeDao;
+import com.model.Code;
+import com.model.User;
+import com.utils.DBConnector;
+import org.apache.log4j.Logger;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Optional;
+
+public class CodeMySQLDaoImpl implements CodeDao {
+
+    private static final Logger logger = Logger.getLogger(CodeMySQLDaoImpl.class);
+
+    @Override
+    public void add(Code code) {
+        String query = String.format("INSERT INTO code (user_id, value) VALUES (%d, '%s')",
```

Comment:
try to make sql queries as constants)

Reply:
I can't do this without preparedStatement )